crates/valgrind/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "wasm-valgrind"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Let's update all the metadata here to match the other crates in `crates/` -- developers as "The Wasmtime Developers", license, current version, etc (and remove the comment from the starter template below).